public/signup.php

```diff
@@ -1,3 +1,13 @@
+<?php
```

Comment:
Hey Fengjun,

What is this php part for? I don't think we need to do a post action in here. I think your modification in the JS file should be enough. What do you think? But otherwise it looks great!! :)